src/main/java/org/broadinstitute/consent/http/resources/DatasetResource.java

```diff
@@ -203,15 +209,16 @@ public Response validateDatasetName(@QueryParam("name") String name) {
     public Response getDataSetSample() {
         String msg = "GETting Data Set Sample";
         logger().debug(msg);
-        String fileName = "DataSetSample.tsv";
+        String dataSetSample = "Dataset Name\tData Type\tSpecies\tPhenotype/Indication\t# of participants\tDescription\tdbGAP\tData Depositor\tPrincipal Investigator(PI)\tSample Collection ID\tConsent ID"
+                + "\n(Bucienne Monco) - Muc-1 Kidney Disease\tDNA, whole genome\thuman\tmuc-1, kidney disease\t31\tmuc-1 patients that developed cancer , 5 weeks after treatment\thttp://....\tJohn Doe\tMark Smith\tSC-20658\t1";
         InputStream inputStream = null;
         try {
-            inputStream = Resources.getResource(fileName).openStream();
+            inputStream = new ByteArrayInputStream(dataSetSample.getBytes());
         } catch (Exception e) {
             logger().error("Error when GETting dataset sample. Cause: " + e);
             return createExceptionResponse(e);
         }
-        return Response.ok(inputStream).header("Content-Disposition", "attachment; filename=" + fileName).build();
+        return Response.ok(inputStream).header("Content-Disposition", "attachment; filename=DataSetSample.tsv").build();
```

Comment:
`DataSetSample.tsv` could also be moved to a class level field so it could be customized by a test.